Sources/Sentry/SentrySDK.m

```diff
@@ -83,6 +83,28 @@ + (void)startWithOptionsObject:(SentryOptions *)options
     [SentrySDK installIntegrations];
 }
 
++ (void)startWithDsn:(NSString *)dsn
+{
+    [self startWithDsn:dsn withConfigureOptionsBlock:^(SentryOptions *options) {}];
+}
+
++ (void)startWithDsn:(NSString *)dsn
+    withConfigureOptionsBlock:(void (^)(SentryOptions *options))configureOptions
+{
+    NSError *error = nil;
+    SentryOptions *options = [[SentryOptions alloc] initWithDict:@{ @"dsn" : dsn }
+                                                didFailWithError:&error];
+    if (nil != error) {
+        [SentryLog logWithMessage:@"Error while initializing the SDK"
+                         andLevel:kSentryLogLevelError];
+        [SentryLog logWithMessage:[NSString stringWithFormat:@"%@", error]
+                         andLevel:kSentryLogLevelError];
```

Comment:
M: what type of errors do we see here? eg a null DSN should crash the app, an empty DSN should disable the SDK, probably all the other errors should be suppressed? right now it'll silently fail and one will need to debug to understand why it's not working.